index.html

```diff
@@ -0,0 +1,137 @@
+                    <h3 class="w-25">Exhibition</h3>
+                    <p class="w-50 text-white">Fugiat quaerat tempore repudiandae expedita doloribus, sit consectetur.
+                    </p>
+                </div>
+                <div class="activity d-flex flex-md-column align-items-center justify-content-between justify-content-md-around">
+                    <img src="img/forum.png" alt="" class="activity-img">
+                    <h3 class="w-25">Forum</h3>
+                    <p class="w-50 text-white">Voluptatem repellat esse neque officiis, vel obcaecati voluptate ipsa.
+                    </p>
+                </div>
+                <div class="activity d-flex flex-md-column align-items-center justify-content-between justify-content-md-around">
+                    <img src="img/workshop.png" alt="" class="activity-img">
+                    <h3 class="w-25">Workshop</h3>
+                    <p class="w-50 text-white">Accusamus sed vel harum totam eveniet, natus ea id perferendis ab
+                        aspernatur.</p>
+                </div>
+                <div class="activity d-flex flex-md-column align-items-center justify-content-between justify-content-md-around">
+                    <img src="img/ignite.png" alt="" class="activity-img">
+                    <h3 class="w-25">CC Ignite</h3>
+                    <p class="w-50 text-white">Accusamus sed vel harum totam eveniet, natus ea id perferendis ab
+                        aspernatur.</p>
+                </div>
+            </div>
+            <a href="#" class="program-link d-none d-md-block">SEE THE WHOLE PROGRAM</a>
+            <button type="button" class="button d-block d-md-none mx-auto mt-4 mb-4">Join WebDev Summit 2022</button>
+        </section>
+        <section class="speakers container-fluid pt-4 pb-4">
+            <h2 class="text-center">Featured Speakers</h2>
+            <hr class="section-hr mx-auto mb-4">
+            <div class="row speakers-wrapper"></div>
+        </section>
+        <section class="partner container-fluid mt-4 pt-4 pb-4">
+            <h2 class="text-white text-center">Partner</h2>
+            <hr class="section-hr mx-auto mb-4">
+            <div class="partner-wrapper d-flex flex-wrap flex-md-nowrap mx-auto">
+                <div class="partner-div w-50 w-md-25">
+                    <img src="img/Mozilla_Foundation_logo_(pre_2017).svg" alt="" class="img-fluid">
+                </div>
+                <div class="partner-div w-50 w-md-25">
+                    <img src="img/Airbnb_Logo_Bélo.svg" alt="" class="img-fluid">
+                </div>
+                <div class="partner-div w-50 w-md-25">
+                    <img src="img/Daum_Kakao_wordmark_-_2014.svg" alt="" class="img-fluid">
+                </div>
+                <div class="partner-div w-50 w-md-25">
+                    <img src="img/Naver_Logotype.svg" alt="" class="img-fluid">
```

Comment:
Kindly, the Alt attributes should contain a description of your image, that will greatly help the accessibility of your website.